Kudu.Services.Web/Content/Scripts/ProcessExplorer.js

```diff
@@ -158,9 +158,17 @@ var Utilities = (function () {
             iframe.style.display = "none";
             document.body.appendChild(iframe);
         }
-        
+        iframe.onload = function (e) {
+            if (showResponseMessage) {
+                var iframeDocument = iframe.contentDocument || iframe.contentWindow.document; // for both IE and other
+                var iFrameBody = iframeDocument.getElementsByTagName('body')[0];
+                if (iFrameBody.innerText && iFrameBody.innerText.length > 0) {
```

Comment:
some of the browser might not support innertext
http://stackoverflow.com/questions/1359469/innertext-works-in-ie-but-not-in-firefox

will be better if you just use JQuery library to get the text